htdocs/core/class/discount.class.php

```diff
@@ -470,6 +473,14 @@ public function link_to_invoice($rowidline, $rowidinvoice)
 				$this->fk_facture_line = $rowidline;
 				$this->fk_facture = $rowidinvoice;
 			}
+			if (!$notrigger) {
+				// Call trigger
+				$result = $this->call_trigger('DISCOUNT_LINK_TO_INVOICE', $user);
```

Comment:
Trigger codes must be limited to the 4 CRUD events. Here we modify the discount line, so you can use the code
DISCOUNT_MODIFY

Reply:
Thanks for the correction!